lms/djangoapps/certificates/management/commands/fix_cert_records.py

```diff
@@ -0,0 +1,54 @@
+"""
+(Temporary) Management command used to fix some incorrectly generated certificate records created as a side effect of
+CR-3792.
+"""
+import datetime
+import logging
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from common.djangoapps.student.models import CourseEnrollment
+from lms.djangoapps.certificates.generation_handler import _generate_certificate_task
+from lms.djangoapps.certificates.models import GeneratedCertificate
+
+log = logging.getLogger(__name__)
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Temporary Management command to fix the incorrect mode on a number of records in the
+    `CERTIFICATES_GENERATEDCERTIFICATE table.
+    """
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-l', '--limit',
+            metavar='LIMIT',
+            dest='limit',
+            help='number of records to process at once'
+        )
+
+    def handle(self, *args, **options):
+        if options.get('limit'):
+            limit = int(options['limit'])
+        else:
+            limit = 1000
+
+        # We started creating the incorrect certificate records around May 10th, 2021.
+        certs = GeneratedCertificate.objects.filter(
+            mode='honor',
+            created_date__gte=datetime.date(2021, 5, 10)
```

Comment:
I just wanted to point out this hard-coded date in the management command. We started seeing records with the mode (incorrectly) set to `honor` starting on May 10th, 2021. They stop being generated on June 22nd, 2021.

I figured this might be okay since I was intending to throw this management command away after the cleanup is completed.